.changes/next-release/feature-189a5640-4426-4440-ae5a-090363c9e50b.json

```diff
@@ -0,0 +1,4 @@
+{
+  "type" : "feature",
+  "description" : "Refactor the AWS explorer's nodes to use an extension point"
```

Comment:
This is a user facing changelog for a not user facing change so maybe something like `which lays the groundwork for enabling wider IDE support in future releases`

Reply:
I'm not sure we need a changelog entry for this.

Reply:
I will delete this entry.